plugins/foreman_maintain/0.2/index.md

```diff
@@ -0,0 +1,405 @@
+---
+layout: plugin
+title: Foreman Maintain 0.2 Manual
+version: 0.2 
+---
+
+# Foreman-maintain
+
+For general information about foreman-maintain, please check the [github repo](https://github.com/theforeman/foreman_maintain)
+
+# Install
+
+To install foreman-maintain on foreman or katello instances, please run:
+```
+# yum install rubygem-foreman_maintain
+```
+
+Foreman-maintain is installed as a dependency in Katello 3.7 and up.
+
+# Service management
+
+Services can be managed using `foreman-maintain service`. This is based on the existing `katello-service` command. For all options, run `foreman-maintain service --help`.
+
+# Backup
+
+### Supported systems
+Backup and Restore in foreman-maintain works on the following:
+* Katello (>3.0)
+* Foreman-proxy with content (>3.0)
+
+Backup and Restore support for standalone Foreman is currently in development.
+
+
+### General
+Note: We assume all the foreman-maintain commands are run as a root user.
+
+Backup utility in `foreman-maintain` offers three backup strategies
+* Offline - The whole Katello instance will be turned off completely for the entire backup.
```

Comment:
I suppose when Foreman support is added we'll come back through and change these from Katello to Foreman? I realize this is currently specific to the katello plugin being available but the instance itself is still Foreman.

Reply:
Yes, this should be updated when Foreman backup/restore support is finished